### ISSUE TITLE
Connect then cancel then refresh always triggers metamask signing

### ISSUE THREAD

**Issue by hiaux0** [PrimeDAO/prime-deals-dapp#813]
STEPS
1. (Clear local storage to be save)
2. Connect to a wallet
3. accept disclaimer
4. cancel metamask
5. (popups and modal appear as expected, just cancel/accept/ignore them)
6. Refresh

RESULT
7. New metamask signing appears (again for every subsequent refresh)

EXPECTED
8. Metamask to not show up once I rejected once.

VIDEO
I refreshed 4 times 

https://user-images.githubusercontent.com/30693990/165488630-c4ce5eeb-ea56-4b25-aaac-196903937fb8.mp4

**Comment by paweljanicki** [comment]
In my opinion, how to handle this is a question for business people @blink-km can you elaborate?

The problem I see:
If user cancel firebase authentication (signature once) and we don't show it to them on page refresh, how are they going to know they are not authenticated and therefore cannot see private deal or vote/edit/etc. their deals? There is nothing indicating that they are not authenticated to our database and that the functionality is limited. There is also no CTA button for them to trigger firebase authentication

**Comment by blink-km** [comment]
I'm leaning towards keeping the behavior this way. I am looping in UX and business perspectives. Will get back to you with a decision soon.

**Comment by dkent600** [comment]
Agreed, should work just like metamask wallet which also does not persist a cancellation

**Comment by blink-km** [comment]
We will keep it this way so the user can find where to authenticate when they want to

**Comment by BartuCulha** [comment]
Just want to readdress this question. 


Can an address just sign once and firebase authentication message wont reappear ever again?

**Comment by BartuCulha** [comment]
it's a very weird UX, i would feel suspicious and annoyed having to sign a lot of times.

**Comment by blink-km** [comment]
https://github.com/primedao/prime-deals-dapp/issues/843 Created a new card for authenticating once
